packages/terra-form-select/src/multiple/Frame.jsx

```diff
@@ -661,8 +664,8 @@ class Frame extends React.Component {
           <span id={labelId}>{this.ariaLabel()}</span>
           <span id={descriptionId}>{this.renderDescriptionText()}</span>
         </div>
-        <div className={cx('display')} aria-label={this.ariaLabel()}>
-          {this.getDisplay(displayId, ariaDescribedBy)}
+        <div className={cx('display')}>
+          {this.getDisplay(displayId, ariaDescribedBy, id)}
```

Comment:
nit: maybe it's worth considering changing the API to accept an object and then de-structure it to extract whatever that is needed inside `getDisplay`.

Reply:
Added id as prop in 5bb136d